### ISSUE TITLE
Responsive grid

### ISSUE THREAD

**Issue by da5nsy** [all-contributors/all-contributors#660]
### Is your feature request related to a problem? Please describe.
I get frustrated that:
- Each commit to add a new contributor is a rather large commit because of the way the HTML is set up ([example](https://github.com/alan-turing-institute/the-turing-way/pull/2764/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5))
- The grid on the readme is not responsive so when it is viewed on a small screen the right-most columns get cut off

### Describe the solution you'd like
A system whereby the grid was made responsive such that when a new user was added, it didn't change the whole grid (and also so that it looks nice).

### Describe alternatives you've considered
We could [turn off alphabetic ordering for our grid](https://github.com/alan-turing-institute/the-turing-way/pull/2681), and that would be a partial solution.

### Additional context
**I've mocked up a hacky idea [here](https://github.com/da5nsy/css-in-readme-like-wat).**
I don't really know HTML/CSS, and so this is probably as far as I can push this, but I think there might be something there that others might be able to use for inspiration. It's based on a combo of [this](https://mastery.games/post/tile-layouts/) and [this](https://github.com/sindresorhus/css-in-readme-like-wat).
_(For some reason, the image links aren't working 🤷 )_

**Comment by Berkmann18** [comment]
One of the issues we have to deal with is that GH doesn't allow any CSS so getting the table to be viewable the way we like has limits.
If you have alphabetical ordering turned off, the spread of changes isn't too big to see exactly what changed in your README.md or whatever file is set in your AC config file.

**Comment by da5nsy** [comment]
Agreed! If you have a look at the mock-up I made you can see my (partially successful) attempt to get around the lack of direct CSS access.

**Comment by Berkmann18** [comment]
How practical is that hack and do you know if the elements used are supported for the foreseeable HTML versions?

**Comment by da5nsy** [comment]
> How practical is that hack

Currently, it is not practical. Perhaps someone with a better grasp of html/css than I could make it work, but I can't fix two major things currently:
- Loading the avatar images through the SVG (the [link](https://avatars3.githubusercontent.com/u/3739866?v=4?s=100) is valid but it [doesn't currently render](https://github.com/da5nsy/css-in-readme-like-wat/blob/main/readme.md)).
- Making the SVG box height [on the readme](https://github.com/da5nsy/css-in-readme-like-wat) be a variable/responsive height - this seems like it should be possible, but I haven't been able to work out a way to do it.

> do you know if the elements used are supported for the foreseeable HTML versions

I do not. The key elements used in this hack are [`foreignObject`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject) and [`grid-template-columns`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns).
Another issue is that github may change the way that readme/markdown files are rendered, which makes the rawest possible HTML more appealing.
